includes/abstracts/abstract-wc-stripe-connect-rest-controller.php

```diff
@@ -4,6 +4,8 @@
 	exit;
 }
 
+// phpcs:disable WordPress.Files.FileName
```

Comment:
Is this really necessary?

Reply:
Yes, because the expected Class Name for that filename would be `Abstract_WC_Stripe_Connect_REST_Controller`